### PR TITLE
Changed to  fontawsome 5 move search to menu

### PR DESCRIPTION
**Pull request by jepp3**

### CODE REVIEW

File: ui/package.json [hunk 1]
```diff
@@ -24,6 +24,7 @@
     "@babel/polyfill": "^7.0.0",
     "@babel/preset-env": "^7.0.0",
     "@babel/preset-react": "^7.0.0",
+    "@fortawesome/fontawesome-free": "^5.9.0",
```
Comment (flowertwig) [inline ui/package.json]: 🏰 ? :)

File: ui/src/components/layout/header.js [hunk 2]
```diff
@@ -73,20 +90,33 @@ class Header extends React.Component {
...
                     </div>
                     <div>
-                        <div className="user" onClick={this.toggleDropdown.bind(this)}>
-                            <Button color="primary" text>
-                                <i className="fa fa-user" /> {this.getUser()}
+                        <div className="item">
+                        <Button border small onClick={this.navigate.bind(this, '/create/question')}>
+                            <i className="fa fa-pen" /> {t`New question`}
+                        </Button>
+                            {/*<Dropdown
```
Comment (flowertwig) [inline ui/src/components/layout/header.js]: Remove?

File: ui/package.json (hunk 1 above)
Comment (jepp3) [reply ui/package.json]: The organization on the Font Awesome GitHub project is in fact called Fort Awesome. That's where the name comes from with just a single letter variation: Font Awesome.

One can read more about the name here: https://www.impresspages.org/awesome-q-a-with-font-awesome-developer-dave-gandy#.XSWsT1UzZhE

File: ui/src/components/layout/header.js (hunk 2 above)
Comment (jepp3) [reply ui/src/components/layout/header.js]: I thought about to remove it, but we had a discussion to leave it as it is, when we started the mvp. We had some functionallity at that time in the ui, that was not supported by the backend.

**Pull request merged**